os/mm/mm_heap/mm_mallinfo.c

```diff
@@ -111,9 +112,19 @@ int mm_mallinfo(FAR struct mm_heap_s *heap, FAR struct mallinfo *info)
 
 		mm_takesemaphore(heap);
 
-		for (node = heap->mm_heapstart[region]; node < heap->mm_heapend[region]; node = (struct mm_allocnode_s *)((char *)node + node->size)) {
+		prev = NULL;
```

Comment:
Duplicated initialization

Reply:
It needs to be initialized every time in the for loop for each memory region. (Line 106)

Reply:
Then we can remove initialization in 90.
```
struct ... *prev;
```
is enough.

Reply:
ok